fdbserver/TagPartitionedLogSystem.actor.cpp

```diff
@@ -515,8 +515,19 @@ Future<Version> TagPartitionedLogSystem::push(Version prevVersion,
 	std::vector<Future<TLogCommitReply>> allReplies;
 	int location = 0;
 	Span span("TPLS:push"_loc, spanContext);
-	int tLogCount = tpcvMap.present() ? tpcvMap.get().size() : 0;
 
+	std::unordered_map<int, int> tLogCount;
+	if (SERVER_KNOBS->ENABLE_VERSION_VECTOR_TLOG_UNICAST) {
+		int location = 0;
+		for (auto& it : tLogs) {
+			for (int loc = 0; loc < it->logServers.size(); loc++) {
+				if (tpcvMap.get().find(location) != tpcvMap.get().end()) {
+					tLogCount[loc]++;
```

Comment:
I'm confused as the meaning of `tLogCount`.

In the HA setting, primary and satellite tlogs could have the same `loc` number, but they are unrelated: for a mutation, the 3 tlogs chosen in the primary can be different `loc` for those in satellite, because satellite use a random log router tag.

Reply:
I will put this into draft mode, and make it public again when I have a fix.